wormhole-connect/src/store/redeem.ts

```diff
@@ -36,7 +39,11 @@ export const redeemSlice = createSlice({
   reducers: {
     setTxDetails: (
       state: RedeemState,
-      { payload }: PayloadAction<ParsedMessage | ParsedRelayerMessage>,
+      {
+        payload,
+      }: PayloadAction<
+        ParsedMessage | ParsedRelayerMessage | UnsignedNttMessage
```

Comment:
same question here. It seems like we're adding route-specific code to a generic function

Reply:
The properties on `UnsignedNttMessage` are needed throughout the codebase for the ntt routes. I'm not really sure what the right solution is here given that `ParsedRelayerMessage` is also route-specific. Maybe we could remove `UnsignedNttMessage` and add an optional `ntt` property to `ParsedMessage` instead, but I feel like that's not really solving the issue at its core